WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderFragment.kt

```diff
@@ -39,6 +39,9 @@ class ReaderFragment : Fragment(R.layout.reader_fragment_layout), ScrollableView
     private var searchMenuItem: MenuItem? = null
     private var settingsMenuItem: MenuItem? = null
 
+    private var _binding: ReaderFragmentLayoutBinding? = null
+    private val binding get() = _binding!!
```

Comment:
I know this is recommended but tbh I don't think it's necessary. I think it's ok to use directly the nullable field. At least the person who uses it will understand the field is alive only when the view is alive (and they can call `!!` consciously)

Reply:
Addressed in b48b55e.